labdao/plex
# LAB-1342 resubmit experiment

## What type of PR is this? 

- 🎮 Feature

## Description

backend logic for flow level aggregation.
this is a prerequisite for the unified view frontend updates. this PR only focuses on the backend logic
this PR also contains changes for [LAB-1408](https://linear.app/convexitylabs/issue/LAB-1408/seq-only-tool-updates)

## Related Tickets & Documents

Unified view linear ticket for reference: [LAB-1400](https://linear.app/convexitylabs/issue/LAB-1400/experiment-inputs-outputs-unified-view)
labsay seq-only tool updates are done in the Convexity repo in this PR: [#30](https://github.com/convexitylabs/convexity/pull/30)

Note: Combinatorial submissions are enabled to be able to test this feature

## Steps to Test

Use tool `labsay seq-only with checkpoints v0.4`
Submit 2 or more jobs combinatorially
Upload pdb files for checkpointing
The tool will generate random numbers for plddt and ipae scores for x and y axis

Expected output: Aggregated checkpoints (3 points per job) all shown together in the scatterplot. Conditions shown below the plot with their respective logs and output files.
